src/home/NotificationCenter.tsx

```diff
@@ -269,9 +269,7 @@ const styles = StyleSheet.create({
     flex: 1,
   },
   title: {
-    ...fontStyles.large600,
-    fontSize: 24,
-    lineHeight: 32,
+    ...typeScale.titleMedium,
```

Comment:
why `titleMedium` here and not `semiBoldLarge` as the others? because of the custom `fontSize` and `lineHeight` we currently have?